README.md

```diff
@@ -98,3 +98,13 @@ all functionality comes from plugins.
 Want to add functionality to run against your entire system, composed of many repos, all 
 at once? Write a plugin! Plugins follow the git-subcommand pattern of commander.js and implement a single index.js function 
 to apply their top level menu to the bare meta command. 
+
+### Available Plugins
+
+* (meta-init)[https://github.com/mateodelnorte/meta-init]
+* (meta-project)[https://github.com/mateodelnorte/meta-project]
+* (meta-git)[https://github.com/mateodelnorte/meta-git]
+* (meta-exec)[https://github.com/mateodelnorte/meta-exec]
+* (meta-gh)[https://github.com/mateodelnorte/meta-gh]
+* (meta-npm)[https://github.com/mateodelnorte/meta-npm]
+* (meta-yarn)[https://github.com/mateodelnorte/meta-yarn]
```

Comment:
Couple o notes. 

meta-exec, meta-init, meta-git, and meta-project are plugins, but are included by default (they're installed globally as dependencies of meta). so perhaps that can be annotated: https://github.com/mateodelnorte/meta/blob/master/package.json#L39-L42

all other plugins are contextual, so they're `npm i` or `yarn add` installed to your local meta repo. we could use some clearer language about how all that works as well. 

meta-gh is currently incomplete. it's a skelaton, but if you follow the lead in meta-git, you can complete it very quickly. I can lend you a hand with that if you're interested.